apps/system/js/attention_window.js

```diff
@@ -107,7 +107,7 @@
   AttentionWindow.REGISTERED_EVENTS =
     ['mozbrowserclose', 'mozbrowsererror', 'mozbrowservisibilitychange',
       'mozbrowserloadend', 'mozbrowserloadstart',
-      '_localized', 'click', '_willdestroy'];
+      '_created', '_localized', 'click', '_willdestroy'];
```

Comment:
If we create audioChannelController inside installSubComponents we don't need this